src/Baka/Traits/SlugTrait.php

```diff
@@ -29,11 +29,13 @@ public static function bootSlugTrait()
     /**
      * Get Model.
      */
-    public static function getBySlug(string $slug, CompanyInterface $company): self
+    public static function getBySlug(string $slug, CompanyInterface $company, int $appId, bool $fail = false): ?self
```

Comment:
@Keysie27 why just not create 2 metho one with and the other with fail similar to what laravel does ?

Reply:
To avoid duplicate code, but following the standard in the last commit the methods were already separated